From 0a253d7c6d33c2449b5a9c2009164cfae0d269a5 Mon Sep 17 00:00:00 2001
From: Andy Grover <agrover@redhat.com>
Date: Tue, 7 Oct 2014 11:38:19 -0700
Subject: [PATCH] iscsiuio: Use attribute(unused) for *icmpv6_hdr

I think this variable serves a documentation and completeness purpose
even though it is not used in the function's computation. In order to
prevent gcc from issuing a warning about set but unused variables,
annotate it with __attribute__((__unused__)).

Signed-off-by: Andy Grover <agrover@redhat.com>
---
 iscsiuio/src/uip/uip.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iscsiuio/src/uip/uip.c b/iscsiuio/src/uip/uip.c
index 703cefc..ec3d6ce 100644
--- a/iscsiuio/src/uip/uip.c
+++ b/iscsiuio/src/uip/uip.c
@@ -1023,7 +1023,7 @@ void uip_process(struct uip_stack *ustack, u8_t flag)
 	struct uip_tcp_ipv4_hdr *tcp_ipv4_hdr = NULL;
 	struct uip_tcp_hdr *tcp_hdr = NULL;
 	struct uip_icmpv4_hdr *icmpv4_hdr = NULL;
-	struct uip_icmpv6_hdr *icmpv6_hdr = NULL;
+	struct uip_icmpv6_hdr *icmpv6_hdr __attribute__((__unused__)) = NULL;
 	struct uip_udp_hdr *udp_hdr = NULL;
 
 	/*  Drop invalid packets */
-- 
2.5.0

